README.md

```diff
@@ -261,13 +261,13 @@ If you want to install [io.js](https://github.com/iojs/io.js/):
 nvm install iojs
 ```
 
-If you want to install a new version of io.js and migrate npm packages from a previous version:
+If you want to install a new version of `io.js` and migrate npm packages from a previous version:
 
 ```sh
 nvm install iojs --reinstall-packages-from=iojs
 ```
 
-The same guidelines mentioned for migrating npm packages in Node.js are applicable to io.js.
+The same guidelines mentioned for migrating npm packages in `Node.js` are applicable to `io.js.
```

Comment:
this is missing the final backtick; but i'm not actually sure why any of these should be backticked - they're the actual names of the software, not the names of a binary or a command.

(also, and i know this was already in master, but "node" should always be lowercased).